src/nimblepkg/init.nim

```diff
@@ -155,6 +156,7 @@ version       = $#
 author        = $#
 description   = $#
 license       = $#
+backend       = $#
```

Comment:
Would be great if you could only add this when a backend that isn't `c` is selected.

Reply:
Sure. How should I do that, use an `if...else` statement with two calls to `writeFile`? I'm not that well-versed in Nim yet, but i feel like it would have a cleaner way to do that.

Reply:
I would create a variable to store the whole `"backend = $#"` string (set it to an empty string if the default backend is selected), then just add it to the final string at the end.

Reply:
Will do. I should have it done in a minute or two.